go/libraries/doltcore/sqle/database.go

```diff
@@ -654,6 +661,9 @@ func (db Database) GetWorkingSet(ctx *sql.Context) (*doltdb.WorkingSet, error) {
 	if !ok {
 		return nil, fmt.Errorf("no root value found in session")
 	}
+	if dbState.WorkingSet == nil {
```

Comment:
Don't know if we actually want this behavior. In general an accessor like this shouldn't second-guess what a client is going to do with the things it returns. It's a valid state for a session to not have a working set, and returning a nil value here is fine, it doesn't need to be an error. Generally speaking we don't want the presence of an error to signal needed changes in business logic, only actual errors that mean something went wrong, which isn't the case here.